### PR TITLE
added moneylover

### PR DESCRIPTION
**Pull request by lacherogwu**
## Description
MoneyLover is a personal finance manager for iOS and Android. It helps you to track your income and expenses, set budgets, and manage your money.
https://moneylover.me/

This extension is able to interact with the application using it's API

## Screencast
moneylover-1
moneylover-2
moneylover-3

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/moneylover/src/view-wallets.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+          <List.Item
+            key={wallet._id}
+            icon={`https://static.moneylover.me/img/icon/${wallet.icon}.png`}
+            title={wallet.name}
+            subtitle={formatBalance(wallet.balance)}
+            actions={
+              <ActionPanel>
+                <Action.Push
+                  icon={{ source: Icon.Pencil, tintColor: Color.Green }}
+                  title="Adjust balance"
```
Comment (pernielsentikaer) [inline extensions/moneylover/src/view-wallets.tsx:83]:
```suggestion
                  title="Adjust Balance"
```

File: extensions/moneylover/src/view-wallets.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    }
+  }
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm
+            icon={{ source: Icon.Checkmark, tintColor: Color.Green }}
+            title="Adjust balance"
```
Comment (pernielsentikaer) [inline extensions/moneylover/src/view-wallets.tsx:150]:
```suggestion
            title="Adjust Balance"
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by lacherogwu** [comment]
@pernielsentikaer Great improvements thanks!
I applied the changes

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋

Looks good to me, approved 🔥

**Pull request merged**